v1.0.1/template/Makefile

```diff
@@ -17,23 +17,28 @@ LDFLAGS:=-g
 RUN:=cmd /C
 
 # Clean up
-RM:=del
+RM:=rm -f
 
 #----------------------------------------------------
 # Files
 #----------------------------------------------------
 # Name of target file
-TARGET:=Testsuite.exe
+TARGET:=Testsuite
 
 # Sourcefiles
 SRCS:=           \
-./System.c       \
```

Comment:
why was `System.c` removed?

Reply:
good question. could not find a reason for that, it was a mistake